sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuAggregateInPandasExec.scala

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
Was this file not moved from the original location of `GpuAggregateInPandasExec`?  This diff implies the history connection between those two files will be lost.

Reply:
sure, moved back